docs/client/request.md

```diff
@@ -211,6 +211,37 @@ Header method returns header value via key, this method will visit all field in
 func (r *Request) Header(key string) []string
 ```
 
+### Headers
+
```

Comment:
Probably also worth adding an example of how to copy the returned value. Is that done through `maps.Collect` too?

It's not clear how would a user copy the values from the iter.Seq2

Reply:
If you do it before closing the request, yes it should be copying it.

Reply:
https://cs.opensource.google/go/go/+/refs/tags/go1.23.4:src/maps/iter.go;l=59